src/platforms/javascript/common/configuration/tree-shaking/index.mdx

```diff
@@ -36,7 +36,49 @@ To make optional code eligible for tree shaking, you can replace various flags i
 : Replacing this flag with `false` will tree shake all code in the SDK that is related to performance monitoring.
 **Attention:** `__SENTRY_TRACING__` must not be replaced with `false` when you're using any performance monitoring-related SDK features (e.g. `Sentry.startTransaction()`). This flag is intended to be used in combination with packages like `@sentry/next` or `@sentry/sveltekit`, which automatically include performance monitoring functionality.
 
-<PlatformSection notSupported={["javascript.nextjs"]}>
+`__RRWEB_EXCLUDE_CANVAS__`
+
+: Replacing this flag with `true` will tree shake all code in the SDK that is related capturing canvas elements with Session Replay. This is only relevant when using [Session Replay](../../session-replay/). You can enable this flag if you don't have any canvas elements on your page you care to record.
```

Comment:
Instead of the relative links we should likely be using platform links here right?

Reply:
but with the relative links, it should work on the platforms, shouldn't it? E.g. if you are in react you should remain in react..?

Reply:
might be a problem with the root JS docs vs guides:

> https://docs.sentry.io/platforms/javascript/session-replay/configuration/

vs

> https://docs.sentry.io/platforms/javascript/guides/angular/session-replay/configuration/

But we can check with the preview deploy (and CI should fail fwiw)

Reply:
Hm, just checked. Seems to work fine so disregard! I could swear at one point these relative links would mess things up and end up linking to the generic guides instead.

Reply:
actually we have a pretty decent link lint step which checks if the links are valid everywhere!

Reply:
(FWIW the `lint:links` command actually found some issue there, mainly that the link does not work for wasm and cordova 😅 fixed that! good CI!)